cli/lib/gather.js

```diff
@@ -259,30 +263,48 @@ class Categorizer {
 					}
 				}
 
-				if (LAUNCH_IMAGE_REGEXP.test(info.name)) {
+				if (this.platform === 'ios' && LAUNCH_IMAGE_REGEXP.test(info.name)) {
 					results.launchImages.set(relPath, info);
 					return;
 				}
 				// }
 				// fall through to lump with JPG...
 
 			case 'jpg':
-				// if the image is the LaunchLogo.png, then let that pass so we can use it
-				// in the LaunchScreen.storyboard
-				const m = info.name.match(LAUNCH_LOGO_REGEXP);
-				if (m) {
-					info.scale = m[1];
-					info.device = m[2];
-					results.launchLogos.set(relPath, info);
 
-				// if we are using app thinning, then don't copy the image, instead mark the
-				// image to be injected into the asset catalog. Also, exclude images that are
-				// managed by their bundles.
-				} else if (this.useAppThinning && !relPath.match(BUNDLE_FILE_REGEXP)) {
-					results.imageAssets.set(relPath, info);
-				} else {
-					results.resourcesToCopy.set(relPath, info);
+				// Toss Android splash screens into launchImages
+				if (this.platform === 'android') {
+					if (info.name === 'default.9' || info.name === 'default') {
```

Comment:
9-patch images have to be PNGs. So, it should look like this...
```javascript
if ((info.name === 'default') || (info.name === 'default.9' && info.ext === 'png')) {
    // ...
}
```

Reply:
There's another issue here.

We only accept "default.png"/"default.9.png" as a splash screen image if it's in the below directories. That is, it should look for the default PNG in the root "src"/"origSrc" given to the `walk()` method.
- `./Resources`
- `./Resources/android`
- `./Resources/images/res-*`
- `./Resources/android/images/res-*`

The issue we have now is that it's wrongly grabbing a "default.png" from any folder which is causing a `EEXISTS: file already exists` build error since it collides with the "default.png" that I have in the correct folder.

Reply:
Also the old handling of "default.png" file collision provided a more meaningful error message. If i add the following files to my project:
- `./Resources/default.png`
- `./Resources/android/default.png`

The I would get the following build error.
```
[ERROR] Found conflicting resources:
[ERROR]    Resources/default.9.png
[ERROR]    Resources/android/default.9.png
[ERROR] You cannot have resources that resolve to the same resource entry name
```

Although I would be okay with us ditching the error and letting the last one win. I think files under the "android" subdirectory should take priority over files in the root directory. Similar to how Android handles "res" file name collision where the **app's** "res" files are used over the **library's** res files of the same name, allowing the app to "override" the library's resources (this is intentional).